src/site/stages/build/process-cms-exports/transformers/helpers.js

```diff
@@ -11,6 +11,15 @@ function getDrupalValue(arr) {
   return null;
 }
 
+function createMetaTag(type, key, value) {
```

Comment:
Is this in more than just `node-page`?

Reply:
Yeah, I will need metatags for the `event` transformer at least:


```json
"entityMetatags": [
    {
      "__typename": "MetaValue",
      "key": "title",
      "value": "Outreach events | Honoring Alaskan Veterans | Veterans Affairs"
    },
    {
      "__typename": "MetaValue",
      "key": "twitter:card",
      "value": "summary_large_image"
    },
    {
      "__typename": "MetaProperty",
      "key": "og:site_name",
      "value": "Veterans Affairs"
    },
    ...
]
```

Reply:
Cool. I ask because I just inlined the meta tags schema for the transformed page node. Seeing this, I'll pull it out into its own schema. :smile: